pkg/operator/ceph/csi/template/rbd/csi-rbdplugin.yaml

```
@@ -106,6 +106,11 @@ spec:
               mountPath: /tmp/csi/keys
             - name: host-run-mount
               mountPath: /run/mount
+            {{ if .EnablePluginSelinuxHostMount }}
+            -  name: etc-selinux
```

Not sure it's a problem, but there is an extra space on these lines
```suggestion
            - name: etc-selinux
```

---

Thanks @travisn , thats been corrected. looks like tests are still failing.. so need to check on it..